src/components/Tags.tsx

```diff
@@ -0,0 +1,25 @@
+import { Chip } from '@mui/material';
+
+interface TagsProps {
+  tags: string[];
+  selectedTags: string[];
+  handleTagClick: (tag: string) => void;
+}
+
+const Tags: React.FC<TagsProps> = ({ tags, selectedTags, handleTagClick }) => {
```

Comment:
The number of tags already clutters the UI. Can you separate the language from the tags in the Filter pane? You can do this in a followup PR.